cmd/cluster-storage-operator-tests-ext/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"os"
+
+	"github.com/openshift-eng/openshift-tests-extension/pkg/cmd"
+	e "github.com/openshift-eng/openshift-tests-extension/pkg/extension"
+	g "github.com/openshift-eng/openshift-tests-extension/pkg/ginkgo"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	utilflag "k8s.io/component-base/cli/flag"
+	"k8s.io/component-base/logs"
+
+	_ "github.com/openshift/cluster-storage-operator/test/e2e"
+)
+
+func main() {
+	logs.InitLogs()
+	defer logs.FlushLogs()
+	pflag.CommandLine.SetNormalizeFunc(utilflag.WordSepNormalizeFunc)
+
+	// Create our registry of openshift-tests extensions
+	extensionRegistry := e.NewRegistry()
+	kubeTestsExtension := e.NewExtension("openshift", "payload", "cluster-storage-operator")
+	extensionRegistry.Register(kubeTestsExtension)
+
+	// Carve up the kube tests into our openshift suites...
+	kubeTestsExtension.AddSuite(e.Suite{
+		Name: "cso/conformance/parallel",
+		Parents: []string{
+			"openshift/conformance/parallel",
+		},
+		Qualifiers: []string{`!labels.exists(l, l == "Serial") && labels.exists(l, l == "Conformance")`},
+	})
+
+	kubeTestsExtension.AddSuite(e.Suite{
+		Name: "cso/conformance/serial",
+		Parents: []string{
+			"openshift/conformance/serial",
+		},
+		Qualifiers: []string{`labels.exists(l, l == "Serial") && labels.exists(l, l == "Conformance")`},
+	})
+
+	// Build our specs from ginkgo
+	specs, err := g.BuildExtensionTestSpecsFromOpenShiftGinkgoSuite()
+	if err != nil {
+		panic(err)
+	}
+
```

Comment:
Found the image registration:
```suggestion

       kubeTestsExtension.RegisterImage(
               e.Image{
                       Index:    0,
                       Registry: "k8s.gcr.io",
                       Name:     "pause",
                       Version:  "3.2",
               })
```

Of course with proper constants, so `storage_performant_policy.go` composes the image name from them.